1. Building a To-Do List Without Framework/submission/index.js

```diff
@@ -104,22 +105,28 @@ const saveTodo = (e) => {
 };
 
 const clearTodo = () => {
-    const clearInfo = `
+        if (todoList.length){
+            const approveClear = confirm("Are you sure you want to delete your all items");
+            if(approveClear){
+                `${localStorage.removeItem("todoList")}
+                ${(todoList = [])}
+                ${renderList()}`
+                const clearInfo = `
                 <div class="text-center alert alert-primary alert-dismissible fade show shadow border-0" role="alert"  data-aos="zoom-in-down">
-                <strong>${
-                    todoList.length > 0
-                        ? "Your items have been successfully removed."
-                        : "Please add any item."
-                }
+                <strong>Your items have been successfully removed.</strong>
                     <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
-                </div>`;
+                </div>`;            
     $("#save-info").html(clearInfo);
-    todoList.length > 0 // if array length is null then do not show confirm msg.
-        ? `${confirm("Are you sure you want to delete your all items")}
-                ${localStorage.removeItem("todoList")}
-                ${(todoList = [])} // array made empty
-                ${renderList()}`
-        : "";
+            }
+        }
+        else{
+            const clearInfo = `
+                <div class="text-center alert alert-primary alert-dismissible fade show shadow border-0" role="alert"  data-aos="zoom-in-down">
+                <strong>Please add any item</strong>
+                    <button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button>
+                </div>`; 
+    $("#save-info").html(clearInfo);
```

Comment:
You should avoid the use of JQuery or any other library for a simple task which can be easily done without it.

Reply:
I used jQuery just to display confirmation messages to the user. Should I remove the display messages and keep it simple?

Reply:
You can do it without jQuery

Reply:
Okay bhaiya give me some time.